src/pages/components/text-input/usage.mdx

```diff
@@ -340,7 +425,21 @@ sections below.
 <Row>
 <Column colLg={12}>
 
-![Text input states](images/text-input-states.png)
+<Tabs>
+
+<Tab label="Fixed">
+
+![Text input states in the fixed style](images/text-input-states.png)
```

Comment:
The image rules in the tabs do not bleed off the image. This also happens on the Style tab as well.  I can see your image files are correct, maybe it has something to do with the tab component.
<img width="954" alt="Screen Shot 2022-09-16 at 2 12 01 PM" src="https://user-images.githubusercontent.com/43969356/190704879-b8a7554e-e56c-470b-b41a-d70acd152c2f.png">

Reply:
Unfortunately, this is because of the tab component, it has a built in border. I don't know how to override it.